src/js/percircle.js

```diff
@@ -123,38 +123,67 @@
                 }, 1000);
             } else if(perdown) {
                 var secs = percircle.attr('data-secs') || options.secs;
-				var timeUpText = percircle.attr('data-timeUpText') || options.timeUpText;
-				if (timeUpText.length > 8) timeUpText='the end';
-                
-				var counter=setInterval(timer, 1000); 
+                var timeUpText = percircle.attr('data-timeUpText') || options.timeUpText;
+                var reset = percircle[0].hasAttribute('data-reset') || options.reset;
```

Comment:
Oh I just took a quick look at it in the morning and I thought you put something like `data-reset="true"` in the html usage example above. Cool!